database/guests.ts

```diff
@@ -1,44 +1,55 @@
-declare module globalThis {
-  let guestsDatabase: Guest[];
-}
-
-// Initialize database to empty array only once
-//
-// TODO: Add a full PostgreSQL database connection
-if (!('guestsDatabase' in globalThis)) {
-  globalThis.guestsDatabase = [];
-}
-
-const guests = globalThis.guestsDatabase;
-
-console.log('guests in db', guests);
-
-export type Guest = {
-  id: number;
-  firstName: string;
-  lastName: string;
-  attending: boolean;
-};
-
-export function getGuests() {
+import { Guest } from '../migrations/00000-createTableGuests';
+import { sql } from './connect';
+
+export const getGuests = async () => {
+  const guests = await sql<Guest[]>`
+    SELECT
+      *
+    FROM
+      guests
+  `;
   return guests;
-}
-
-export function getGuest(id: number) {
-  return guests.find((guest: Guest) => Number(guest.id) === Number(id));
-}
+};
 
-export function addGuest(guest: Guest) {
-  guests.push(guest);
-  return guests;
-}
+export const getGuest = async (id: number) => {
```

Comment:
looks like we're not enforcing the database naming patterns - try out the preview package published by CodeSandbox CI of the new version of the ESLint config here:

-

Reply:
Changed function name to insecure

Reply:
Removed the `insecure` prefix, since we are are now using `auth` with `sessionToken` [848a6ed](https://github.com/upleveled/courses/pull/2546/commits/848a6eddd6f441b89e07ab933660cb3786822798)